presto-singlestore/src/test/java/com/facebook/presto/plugin/singlestore/SingleStoreContainer.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.plugin.singlestore;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.utility.DockerImageName;
+
+public class SingleStoreContainer
+        extends JdbcDatabaseContainer<SingleStoreContainer>
+{
+    private static final String DEFAULT_DOCKER_IMAGE_NAME = "ghcr.io/singlestore-labs/singlestoredb-dev:latest";
```

Comment:
I'm wondering if this should be versioned and updated periodically to protect the CI against backward incompatible changes, Docker image problems or reliability issues.  What are your thoughts?

Reply:
Makes sense